schema.graphql

```diff
@@ -1,3 +1,59 @@
+"""
+Event name. Squid tracks next on-chain events:
+  - "Posts.PostCreated",
+  - "Posts.PostUpdated",
+  - "Posts.PostMoved",
+  - "Reactions.PostReactionCreated",
+  - "Reactions.PostReactionUpdated",
+  - "Reactions.PostReactionDeleted",
+  - "Spaces.SpaceCreated",
+  - "Spaces.SpaceUpdated",
+  - "Profiles.ProfileUpdated",
+  - "SpaceFollows.SpaceFollowed",
+  - "SpaceFollows.SpaceUnfollowed",
+  - "SpaceOwnership.SpaceOwnershipTransferAccepted",
+  - "AccountFollows.AccountFollowed",
+  - "AccountFollows.AccountUnfollowed"
+
+But some of on-chain events have multiple logical meanings.
+As result squid uses synthetic events in addition to native on-chain events.
+Full list of on-squid events is provided below:
```

Comment:
I've used `on-squid` naming as it's similar with `on-chain`. I hope it will be clear.
Maybe we could use smt like `All available in the squid events are:`?